pkg/scheduler/api/types.go

```diff
@@ -114,6 +114,14 @@ type PriorityArgument struct {
 	LabelPreference *LabelPreference
 	// The RequestedToCapacityRatio priority function is parametrized with function shape.
 	RequestedToCapacityRatioArguments *RequestedToCapacityRatioArguments
+	// It uses a scarce resource name to identify the scarce resource
+	ScarceResourceBinPacking *ScarceResourceBinPacking
+}
+
+// ScarceResourceBinPacking holds the scarce resources that are used to configure the corresponding priority function
+type ScarceResourceBinPacking struct {
+	// Used to identify node "groups"
+	ScarceResource string
```

Comment:
what identifies as a scarce resource ? Can i put any extended resource as scarce  ? Where is that configured ? It would be good to put that in comments somewhere

Reply:
Hey yes, you can add any extended resource as scarce, I have mentioned that in notes for reviewers above. Also, I will add a few comments but still looking for a better place to add how to use it. maybe a release note